tests/experimental/metric_repository/test_column_descriptive_metrics_metric_retriever.py

```diff
@@ -58,8 +58,11 @@ def test_get_metrics():
     mock_batch_request = Mock(spec=BatchRequest)
 
     with mock.patch(
-        f"{ColumnDomainBuilder.__module__}.{ColumnDomainBuilder.__name__}.get_effective_column_names",
+        f"{ColumnDescriptiveMetricsMetricRetriever.__module__}.{ColumnDescriptiveMetricsMetricRetriever.__name__}._get_numeric_column_names",
```

Comment:
Note that there is an upcoming PR to refactor these tests and remove some duplication via fixtures: https://github.com/great-expectations/great_expectations/pull/9158